app.rb

```diff
@@ -0,0 +1,177 @@
+require_relative 'classroom'
+require_relative 'person'
+require_relative 'student'
+require_relative 'teacher'
+require_relative 'book'
+require_relative 'rental'
+
+class App
+  def initialize
+    @people = []
+    @books = []
+    @rentals = []
+  end
+
+  def display_menu
+    puts 'Welcome to the School Library App'
+    puts 'Please choose an option by entering a number:'
+    puts '1. List all books'
+    puts '2. List all people'
+    puts '3. Create a person'
+    puts '4. Create a book'
+    puts '5. Create a rental'
+    puts '6. List all rentals for a given person id'
+    puts '7. Exit'
+  end
```

Comment:
Great job team 🤝
- As stated in the project requirements, the role of `main.rb` is only to start your code and ask the user for the initial choice of options (or to quit). Kindly move this piece of code to your `main.rb` file.